app/test_main.py

```diff
@@ -1,3 +1,25 @@
 from app.main import get_human_age
 
-# write your code here
+
```

Comment:
Consider using `@pytest.mark.parametrize` to reduce redundancy in your test cases. This will help in testing multiple inputs with a single test function.